client/app/screens/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Login from 'react-native-login';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Text, View, StyleSheet } from 'react-native';
+import { Body, Button, Col, Container, Content } from 'native-base';
+
+const styles = StyleSheet.create({
```

Comment:
We don't actually need `StyleSheet.create()` with native-base - a plain object is fine.